python/metatensor-torch/metatensor/torch/atomistic/model.py

```diff
@@ -490,7 +490,8 @@ def save(self, file: str, collect_extensions: Optional[str] = None):
                 "metatensor-version": metatensor_version,
                 "extensions": json.dumps(extensions),
                 "extensions-deps": json.dumps(deps),
-                "metadata": json.dumps(export_metadata),
+                "export-metadata": json.dumps(export_metadata),
+                "model-metadata": json.dumps(self._metadata.to_json()),
```

Comment:
`to_json` will produce a string, not a dict, so `json.dumps` will again escape this string
```suggestion
                "model-metadata": self._metadata.to_json(),
```